libs/test-utils/src/console.test.ts

```diff
@@ -1,41 +1,31 @@
 /* eslint-disable no-console */
+import { expect, test } from 'vitest';
 import { suppressingConsoleOutput } from './console';
 
 test.each(['log', 'warn', 'error'] as const)(
   'suppressingConsoleOutput replaces console.%s with a mock',
-  (method) => {
-    suppressingConsoleOutput(() => {
+  async (method) => {
+    const original = console[method];
+    await suppressingConsoleOutput(() => {
       console[method]('test');
       expect(console[method]).toHaveBeenNthCalledWith(1, 'test');
     });
-    expect('mock' in console[method]).toEqual(false);
+    expect(console[method]).toStrictEqual(original);
   }
 );
 
-test('suppressingConsoleOutput returns the value returned by the callback', () => {
-  expect(suppressingConsoleOutput(() => 'test')).toEqual('test');
-  expect('mock' in console.log).toEqual(false);
+test('suppressingConsoleOutput returns the value returned by the callback', async () => {
+  expect(await suppressingConsoleOutput(() => 'test')).toEqual('test');
 });
 
 test('suppressingConsoleOutput resolves to the value returned by the callback', async () => {
   await expect(
     suppressingConsoleOutput(async () => await Promise.resolve('test'))
   ).resolves.toEqual('test');
-  expect('mock' in console.log).toEqual(false);
-});
-
-test('suppressingConsoleOutput throws the error thrown by the callback', () => {
```

Comment:
Now that `suppressingConsoleOutput` always returns a `Promise`, this test is redundant.